src/i18n/en.json

```diff
@@ -59,22 +61,23 @@
   "components.customers.CustomerInfo.contactLanguage_en": "English",
   "components.customers.CustomerInfo.contactLanguage_fi": "Finnish",
   "components.customers.CustomerInfo.contactLanguage_sv": "Swedish",
-  "components.customers.CustomerInfo.customerHasHasoOwnership": "Customer has HASO apartment",
+  "components.customers.CustomerInfo.customerHasHasoOwnership": "Is changing HASO apartment",
```

Comment:
Hmm, is this an intentional change?

Reply:
Yes, the same updated translation string has been used in the new component as well